__tests__/app.test.js

```diff
@@ -237,3 +237,53 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("should return an array", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(Array.isArray(body.comments)).toBe(true);
+      });
+  });
+  it("should return an array of comment objects with the correct keys", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(Array.isArray(body.comments)).toBe(true);
+        body.comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              author: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  it("should return a 400 - bad request error if the id provided is invalid", () => {
+    return request(app)
+      .get("/api/articles/two/comments")
+      .then((response) => {
+        const { body } = response;
+        expect(body.status).toBe(400);
+        expect(body.msg).toBe("bad request - invalid id");
+      });
+  });
+  it("should return 200 - no comments found - if the provided id is valid but has no comments associated with it", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .then((response) => {
+        const { body } = response;
+        expect(response.statusCode).toBe(200);
+        expect(body.comments).toEqual([]);
+      });
+  });
```

Comment:
One more test Aaron, what if there are no comments because that article does not exist in the database?